src/components/Experiment/QuickOutput/Outputs/TextToVideo/TextToVideoOutput.scss

```diff
@@ -0,0 +1,8 @@
+.text-to-video-output {
+  display: block !important;
+}
+
+.video-container div {
```

Comment:
Generally I'd say to try and use classes to apply styling, rather than applying it to all `div`s. Is there a way you can refactor this?

Reply:
Sure, I will try

Reply:
Resolved